spec/index.bs

```diff
@@ -5275,8 +5282,8 @@ called the compute pass encoder can no longer be used.
                         Issue: Add remaining validation.
                     </div>
 
-                1. Push each command in |this|.{{GPUProgrammablePassEncoder/[[command_list]]}} onto the end of
-                    |this|.{{GPUProgrammablePassEncoder/[[command_encoder]]}}.{{GPUCommandEncoder/[[command_list]]}}.
+                1. [=list/Extend=] |this|.{{GPUProgrammablePassEncoder/[[command_encoder]]}}.{{GPUCommandEncoder/[[command_list]]}}
+                    with |this|.{{GPUProgrammablePassEncoder/[[command_list]]}}.
```

Comment:
Maybe if we use this list-of-closures model, we should record commands directly into the encoder's command list instead of first recording it into the pass and then moving it out?

Reply:
I wondered about that, but worried it would lead to ordering issues. I forgot that you can't add any commands to the encoder until any passes are closed, though, so it shouldn't be an issue.

Reply:
Updated this to place the command directly on the encoder's list.